Code/Tools/FontImporter/FontImporter.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <Foundation/Basics.h>
+#include <Foundation/Strings/String.h>
+#include <FontImportOptions.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
```

Comment:
I'm still wondering how this #include works. What exactly is FT_FREETYPE_H and where does it come from?